### ISSUE TITLE
Displaying city names in books template

### ISSUE THREAD

**Issue by Popolechien** [openzim/mwoffliner#1126]
`Issue opened by` @Ubuntu-2020 `and moved from [desktop/448](https://github.com/kiwix/kiwix-desktop/issues/448).`

I don't know how Kiwix displays books in other Wikipedia, but books in Russian Wikipedia in Kiwix without a hint about the city!
This is difficult to explain in words, especially when you do not know English well.
I would like to attach an image, but I don't know how.
Please tell me how to attach an image to a message?
I'll try to explain it in words.
Russian Wikipedia has a [template](https://ru.wikipedia.org/wiki/Категория:Шаблоны:Сокращения_городов_в_библиографии) with abbreviated names of cities.
For example, let's open the [Literature section](https://ru.wikipedia.org/wiki/Амазонка#Литература) in the article about the [Amazon river](https://ru.wikipedia.org/wiki/Амазонка).
Hover the cursor over the letter **М.** or the letter **Л.** and see popup hints: **Москва** and **Ленинград**.
But if you open this article in Kiwix, we will see the usual letters there: **М.** and **Л.**
In Other words, Kiwix doesn't display suggestions for abbreviated city names!
I hope I was able to explain.
If it is possible, make Kiwix display suggestions when you hover the cursor over the abbreviated city name in the Book template.

**Comment by Popolechien** [comment]
Looks like a Javascript thing that activates when hovering on a letter/symbol to display the extended abbreviation.

**Comment by kelson42** [comment]
This is not rendered at all on mobile see https://ru.wikipedia.org/api/rest_v1/page/mobile-html/%D0%90%D0%BC%D0%B0%D0%B7%D0%BE%D0%BD%D0%BA%D0%B0. Probably because this does not make sense on a mobile device. Won't fix.

**Comment by Ubuntu-2020** [comment]
The top of the image shows the section with **Literature** (Литература) and **Links** (Ссылки) in the **Russian Wikipedia** looks like.
The image below shows the same section with **Literature** looks like in **Kiwix**.
To avoid creating multiple images, I created one [**summing image**](https://img11.lostpic.net/2020/05/20/14d36ea4c0dea5daf5774b04ff2c2c0c.png) in GIMP.
In Kiwix when you hover the mouse cursor over the capital letter of the city (**М.** or **Л.**), nothing happens at all.

PS: The image also shows what happens (in **Wikipedia** and in **Kiwix**) when the mouse cursor hovers over a link written in a foreign language (English, German, Swedish, etc.)
